bridges/icon/bridge/bridge.py

```diff
@@ -0,0 +1,302 @@
+from iconservice import *
+from .utils import *
+from .pyobi import *
+
+TAG = "BRIDGE"
+
+
+class BRIDGE(IconScoreBase):
+    def __init__(self, db: IconScoreDatabase) -> None:
+        super().__init__(db)
+        # a single integer
+        self.total_validator_power = VarDB("total_validator_power", db, value_type=int)
+
+        # pubkey:bytes => voting_power:int
+        self.validator_powers = DictDB("validator_powers", db, value_type=int)
+
+        # block_number:int => oracle_state_hash:bytes
+        self.oracle_state = DictDB("oracle_state", db, value_type=bytes)
+
+        # encoded_request:bytes => response:dict
```

Comment:
It seems to be bytes of obi encoded response packet.

Reply:
done